src/debugAdapter/goDebug.ts

```diff
@@ -209,6 +209,7 @@ interface LaunchRequestArguments extends DebugProtocol.LaunchRequestArguments {
 	dlvToolPath: string;
 	/** Delve Version */
 	apiVersion: number;
+	currentFile: string;
```

Comment:
@ramya-rao-a This was the only way to pass along `currentFile`, but this is obviously not configuration. Is it OK to not have `currentFile` in the schema in `package.json`?

Reply:
Yes, its ok. In this case, this is an internal implementation. Current file can always be determined by VS Code using the `vscode.window.activeTextEditor`. It need not be provided by the user